gradle/dependency-locks/testCompile.lockfile

```diff
@@ -33,28 +29,10 @@ com.typesafe:config:1.2.1
 commons-cli:commons-cli:1.4
 commons-codec:commons-codec:1.10
 commons-io:commons-io:2.6
-commons-lang:commons-lang:2.6
 commons-logging:commons-logging:1.2
 io.github.bonigarcia:webdrivermanager:1.7.1
-io.netty:netty-buffer:4.1.31.Final
-io.netty:netty-codec-http:4.1.31.Final
-io.netty:netty-codec-socks:4.1.31.Final
-io.netty:netty-codec:4.1.31.Final
-io.netty:netty-common:4.1.31.Final
-io.netty:netty-handler-proxy:4.1.31.Final
-io.netty:netty-handler:4.1.31.Final
-io.netty:netty-resolver:4.1.31.Final
-io.netty:netty-transport-native-epoll:4.1.31.Final
-io.netty:netty-transport-native-kqueue:4.1.31.Final
-io.netty:netty-transport-native-unix-common:4.1.31.Final
-io.netty:netty-transport:4.1.31.Final
-javax.activation:javax.activation-api:1.2.0
-javax.annotation:javax.annotation-api:1.3.2
```

Comment:
These big swaths of removed transitive dependencies are the fruit of:
- dependency removal (do we really need Guava just for one trivial class like `ThreadFactoryBuilder`?)
- proper dependency scope usage, like switching from `api` (Maven `compile`) scope to `implementation` (Maven `runtime`)

And having fewer dependencies is always a nice thing:
- shorter classpaths
- less IDE indexing
- smaller shadow JARs
- fewer chances for bugs